### PR TITLE
dxPivotGrid: support SSR

### PR DESCRIPTION
**Pull request by AntonSermyazhko**

### CODE REVIEW

File: js/ui/pivot_grid/ui.pivot_grid.js [hunk 1]
```diff
@@ -1656,7 +1657,9 @@ var PivotGrid = Widget.inherit({
         if(that._needDelayResizing(dataArea.getData()) && isFirstDrawing) {
             setTimeout(updateHandler);
         } else {
```
Comment (DokaRus) [inline js/ui/pivot_grid/ui.pivot_grid.js]: `} else if(hasWindow) {`

File: js/ui/pivot_grid/ui.pivot_grid.js [hunk 2]
```diff
@@ -1656,7 +1657,9 @@ var PivotGrid = Widget.inherit({
...
-            updateHandler();
+            if(hasWindow) {
```
Comment (DokaRus) [inline js/ui/pivot_grid/ui.pivot_grid.js]: `} else if(hasWindow) {`

File: testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.markup.tests.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+"use strict";
```
Comment (nightskylark) [inline testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.markup.tests.js:1]: Is it possible to reuse some of existing PivotGrid tests?

Comment (AntonSermyazhko) [reply testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.markup.tests.js:1]: PivotGrid with SSR creates empty PivotGrid.
I've examined tests and not found tests for empty markup. All of them with data and check functionality.

Comment (nightskylark) [reply testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.markup.tests.js:1]: Oh, right.

**Review by nightskylark — DISMISSED** [review]

File: js/ui/pivot_grid/ui.pivot_grid.js [hunk 4]
```diff
@@ -1656,7 +1657,9 @@ var PivotGrid = Widget.inherit({
         if(that._needDelayResizing(dataArea.getData()) && isFirstDrawing) {
```
Comment (roman-simionov) [inline js/ui/pivot_grid/ui.pivot_grid.js]: Why do you not check hasWindow in this case?

Comment (AntonSermyazhko) [reply js/ui/pivot_grid/ui.pivot_grid.js]: You're right! I will fix it soon.

**Review by DokaRus — COMMENTED** [review]
One more thing - don't forget to remove [this string](https://github.com/AntonSermyazhko/DevExtreme/blob/pivotGrid_SSR/testing/tests/DevExpress.serverSide/widgetsCreation.tests.js#L96) and check that PivotGrid's optionChanged tests works fine

**Review by vconst — APPROVED** [review]

**Pull request merged**